src/mod_bosh.erl

```diff
@@ -161,7 +161,7 @@ node_cleanup(Acc, Node) ->
 -type option() :: {atom(), any()}.
 -spec init(req(), _Opts :: [option()]) -> {cowboy_loop, req(), rstate()}.
 init(Req, _Opts) ->
-    ?DEBUG("issue=bosh_init", []),
+    ?LOG_DEBUG(#{issue => bosh_init}),
```

Comment:
I think we should log `req` field in this module where is possible. So we can format it later in logs.